e2e/wdio/platform/tests/upload-collection.e2e-spec.ts

```diff
@@ -0,0 +1,164 @@
+        checkAbleToSearch(defaultExample);
+        checkAbleToSearch(disableExample);
+        checkAbleToSearch(readonlyExample);
+    });
+
+
+    it('should check orientation', () => {
+        uploadCollectionPage.checkRtlSwitch();
+    });
+
+    function checkDisplayedItemsPerPage(selector: string): void {
+        scrollIntoView(selector + buttons);
+        const tenItems = getElementArrayLength(selector + tableItems);
+        expect(tenItems).toEqual(10);
+
+        click(selector + buttons);
+        click(menuItem);
+
+        const fiveItems = getElementArrayLength(selector + tableItems);
+        expect(fiveItems).toEqual(5);
+
+        click(selector + buttons);
+        click(menuItem, 2);
+
+        const fifteenItems = getElementArrayLength(selector + tableItems);
+        expect(fifteenItems).toEqual(15);
+
+        scrollIntoView(selector + buttons);
+        click(selector + buttons);
+        click(menuItem, 3);
+
+        const twentyItems = getElementArrayLength(selector + tableItems);
+        expect(twentyItems).toEqual(20);
+    }
+
+    function checkSelectedPages(selector: string): void {
+        scrollIntoView(selector + tablePages);
+        const linksLength = getElementArrayLength(selector + tablePages);
+        for (let i = 0; i < linksLength; i++) {
+            click(selector + tablePages, i);
+            expect(getText(selector + tableResult)).toBe(testArr[i]);
+        }
+    }
+
+    function checkSelectedPagesByNextPrevious(selector: string): void {
+        scrollIntoView(selector + linkNext);
+        click(selector + linkNext);
+        expect(getText(selector + tableResult)).toBe(testArr[1]);
+
+        click(selector + linkPrevious);
+        expect(getText(selector + tableResult)).toBe(testArr[0]);
+    }
+
+    function checkColumnHeaderText(selector: string, str: string = ''): void {
+        scrollIntoView(selector + columnHeaders + str);
+        const headerLength = getElementArrayLength(selector + columnHeaders + str);
+        for (let i = 0; i < headerLength; i++) {
+            expect(getText(selector + columnHeaders + str, i)).toBe(columnHeaderTestArr[i]);
+        }
+    }
+
+    function checkAbleToSearch(selector: string): void {
+        scrollIntoView(selector + inputFields);
+        setValue(selector + inputFields, 'Folder');
+        const contentLength = getElementArrayLength(selector + tableContent);
+        expect(contentLength).toEqual(4);
+        for (let i = 0; i < contentLength; i++) {
+            expect(getText(selector + tableContent, i)).toBe(testFolderArr[i]);
+        }
+    }
+
+    function creatingFolder(selector: string): void {
+        scrollIntoView(selector);
+        expect(getText(selector + tableItemCount)).toBe('54');
+        click(selector + transparentButton);
+        setValue(dialogInputField, testFolder1);
+        expect(isElementClickable(dialogCreateButton, 1)).toBe(true, '"Cancel" button not clickable');
```

Comment:
why do we need this assertion here? maybe another test case could check clickability of buttons. I think this should only check the folder creation.